app/src/main/java/com/eveningoutpost/dexdrip/Models/TransmitterData.java

```diff
@@ -17,16 +19,16 @@
 
 @Table(name = "TransmitterData", id = BaseColumns._ID)
 public class TransmitterData extends Model {
-    private final static String TAG = BgReading.class.getSimpleName();
+    private final static String TAG = TransmitterData.class.getSimpleName();
 
     @Column(name = "timestamp", index = true)
     public long timestamp;
 
     @Column(name = "raw_data")
     public double raw_data;
-//
-//    @Column(name = "filtered_data")
-//    public double filtered_data;
+
+    @Column(name = "filtered_data")
```

Comment:
Someone has already created a migration with filtered_data in it in main. The previous branch would not build on the latest main with the migration I had in place. Not sure why the type is real in the migration rather than long. 

 
John Stevens
M - +61 418526349

> On 19 May 2015, at 23:57, Stephen Black <notifications@github.com> wrote:
> 
> In app/src/main/java/com/eveningoutpost/dexdrip/Models/TransmitterData.java:
> 
> >  
> >      @Column(name = "timestamp", index = true)
> >      public long timestamp;
> >  
> >      @Column(name = "raw_data")
> >      public double raw_data;
> > -//
> > -//    @Column(name = "filtered_data")
> > -//    public double filtered_data;
> > +
> > +    @Column(name = "filtered_data")
> this will cause anyone attempting to update their app to crash on all new data unless it includes a migration, its already included in the other dexbridge branch
> 
> —
> Reply to this email directly or view it on GitHub.
>